tests/manual/monkey/src/com/oracle/tools/fx/monkey/util/FX.java

```diff
@@ -0,0 +1,156 @@
+        MenuItem mi = new MenuItem(text);
+        applyMnemonic(mi);
+        mi.setOnAction((ev) -> action.run());
+        lastMenu(b).getItems().add(mi);
+        return mi;
+    }
+
+    public static MenuItem item(MenuBar b, MenuItem mi) {
+        applyMnemonic(mi);
+        lastMenu(b).getItems().add(mi);
+        return mi;
+    }
+
+    public static MenuItem item(MenuBar b, String text) {
+        MenuItem mi = new MenuItem(text);
+        applyMnemonic(mi);
+        lastMenu(b).getItems().add(mi);
+        return mi;
+    }
+
+    private static void applyMnemonic(MenuItem m) {
+        String text = m.getText();
+        if (text != null) {
+            if (text.contains("_")) {
+                m.setMnemonicParsing(true);
+            }
+        }
+    }
+
+    private static Menu lastMenu(MenuBar b) {
+        List<Menu> ms = b.getMenus();
+        return ms.get(ms.size() - 1);
+    }
+
+    public static SeparatorMenuItem separator(MenuBar b) {
+        SeparatorMenuItem s = new SeparatorMenuItem();
+        lastMenu(b).getItems().add(s);
+        return s;
+    }
+
+    public static RadioMenuItem radio(MenuBar b, String text, KeyCombination accelerator, ToggleGroup g) {
+        RadioMenuItem mi = new RadioMenuItem(text);
+        mi.setAccelerator(accelerator);
+        mi.setToggleGroup(g);
+        lastMenu(b).getItems().add(mi);
+        return mi;
+    }
+
+    public static void add(GridPane p, Node n, int col, int row) {
+        p.getChildren().add(n);
+        GridPane.setConstraints(n, col, row);
+    }
+
+    public static <T> void select(ComboBox<T> cb, T value) {
+        cb.getSelectionModel().select(value);
+    }
+
+    public static <T> void selectFirst(ComboBox<T> cb) {
+        cb.getSelectionModel().selectFirst();
+    }
+
+    public static <T> T getSelectedItem(ComboBox<T> cb) {
+        return cb.getSelectionModel().getSelectedItem();
+    }
+
+    public static Window getParentWindow(Object nodeOrWindow) {
+        if (nodeOrWindow == null) {
+            return null;
+        } else if (nodeOrWindow instanceof Window w) {
+            return w;
+        } else if (nodeOrWindow instanceof Node n) {
+            Scene s = n.getScene();
+            if (s != null) {
+                return s.getWindow();
+            }
+            return null;
+        } else {
+            throw new Error("Node or Window only");
+        }
+    }
+
+    /** cascades the window relative to its owner, if any */
+    public static void cascade(Stage w) {
+        if (w != null) {
+            Window p = w.getOwner();
+            if (p != null) {
+                double x = p.getX();
+                double y = p.getY();
+                double off = 20;
+                w.setX(x + off);
+                w.setY(y + off);
+            }
+        }
+    }
+
+    /** creates a Background */
+    public static Background background(Color c) {
+        return new Background(new BackgroundFill(c, null, null));
+    }
```

Comment:
Btw, we already have a utility method that does this. See [Background::fill](https://github.com/openjdk/jfx/blob/jfx20/modules/javafx.graphics/src/main/java/javafx/scene/layout/Background.java#L359).